docs/_includes/segmented-button.html

```diff
@@ -0,0 +1,122 @@
+<div id="segmented-button">
+    {% include section-header.html name="segmented-button" version=page.versions.segmentedButton %}
```

Comment:
Should this be called "Segmented Button?" It's a set of buttons. One of these might be more appropriate:
* Segmented Buttons
* Segmented Button Set
* Segmented Button Group

Reply:
We discussed with design about this on planning. They seemed to want to keep this name to be consistent. 
So I think it's fine

Reply:
Maybe using the plural then? Having a component called "Segmented Button" containing multiple buttons is very odd IMO.

Reply:
I'm starting to err towards plural though because I see this pattern as more like tabs & breadcrumbs (also named plural) because  you never have a single tab or a single breadcrumb.